pkg/apiserver/rest/dualwriter_mode2.go

```diff
@@ -38,7 +39,7 @@ func (d *DualWriterMode2) Mode() DualWriterMode {
 // Create overrides the behavior of the generic DualWriter and writes to LegacyStorage and Storage.
 func (d *DualWriterMode2) Create(ctx context.Context, original runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
 	var method = "create"
-	log := d.Log.WithValues("kind", options.Kind, "method", method)
+	log := d.Log.WithValues("kind", options.Kind, "method", method, "mode", mode2Str)
```

Comment:
any reason we don't just add `.WithValues("mode", mode2Str)` to line 31?

Reply:
nop, that would be a good idea.